ArcGISRuntimeSDKQt_SampleViewers/ArcGISRuntimeSDKQt_Samples/qml/CategoryCard.qml

```diff
@@ -95,6 +96,7 @@ Component {
             sampleListView.currentCategory = displayName;
             SampleManager.currentCategory = SampleManager.categories.get(index);
             stackView.push(sampleListView);
+            GAnalytics.postEvent("category_selected", {"category name": displayName});
```

Comment:
🔧 Here are some more cases of magic strings that we should consider centralizing in some way. There are other cases in the other QML files as well